<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SensorsWidget</class>
 <widget class="QQuickWidget" name="SensorsWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Sensors</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout" stretch="0,1,0">
   <property name="spacing">
    <number>2</number>
   </property>
   <property name="margin">
    <number>6</number>
   </property>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout" stretch="0,1,1,0">
     <property name="topMargin">
      <number>0</number>
     </property>
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string/>
       </property>
       <property name="pixmap">
        <pixmap resource="../rc/icons.qrc">:/icons/bullet_red.png</pixmap>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="plot1">
       <property name="font">
        <font>
         <pointsize>33</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <item>
        <property name="text">
         <string>Analog 0</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Analog 1</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Analog 2</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Analog 3</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Analog 4</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Analog 5</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Back EMF 0</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Back EMF 1</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Back EMF 2</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Back EMF 3</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Accel X</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Accel Y</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Accel Z</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Gyro X</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Gyro Y</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Gyro Z</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Magneto X</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Magneto Y</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Magneto Z</string>
        </property>
       </item>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="plot2">
       <property name="font">
        <font>
         <pointsize>33</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <item>
        <property name="text">
         <string>Analog 0</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Analog 1</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Analog 2</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Analog 3</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Analog 4</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Analog 5</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Back EMF 0</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Back EMF 1</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Back EMF 2</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Back EMF 3</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Accel X</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Accel Y</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Accel Z</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Gyro X</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Gyro Y</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Gyro Z</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Magneto X</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Magneto Y</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Magneto Z</string>
        </property>
       </item>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string/>
       </property>
       <property name="pixmap">
        <pixmap resource="../rc/icons.qrc">:/icons/bullet_blue.png</pixmap>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="PlotWidget" name="plot" native="true">
     <property name="font">
      <font>
       <pointsize>22</pointsize>
      </font>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3" stretch="0,1,1,0">
     <property name="topMargin">
      <number>0</number>
     </property>
     <item>
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string/>
       </property>
       <property name="pixmap">
        <pixmap resource="../rc/icons.qrc">:/icons/bullet_red.png</pixmap>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="val1">
       <property name="font">
        <font>
         <pointsize>33</pointsize>
        </font>
       </property>
       <property name="text">
        <string>512</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="val2">
       <property name="font">
        <font>
         <pointsize>33</pointsize>
        </font>
       </property>
       <property name="text">
        <string>512</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string/>
       </property>
       <property name="pixmap">
        <pixmap resource="../rc/icons.qrc">:/icons/bullet_blue.png</pixmap>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>PlotWidget</class>
   <extends>QQuickWidget</extends>
   <header>PlotWidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../rc/icons.qrc"/>
 </resources>
 <connections/>
</ui>
